tools/odbc/prepared.cpp

```diff
@@ -109,7 +96,7 @@ SQLRETURN SQLNumParams(SQLHSTMT StatementHandle, SQLSMALLINT *ParameterCountPtr)
 SQLRETURN SQLBindCol(SQLHSTMT StatementHandle, SQLUSMALLINT ColumnNumber, SQLSMALLINT TargetType,
                      SQLPOINTER TargetValuePtr, SQLLEN BufferLength, SQLLEN *StrLen_or_IndPtr) {
 	return WithStatementPrepared(StatementHandle, [&](OdbcHandleStmt *stmt) {
-		auto col_nr_internal = ColumnNumber - 1;
+		size_t col_nr_internal = ColumnNumber - 1;
```

Comment:
any reason `auto` does not work?

Reply:
It's just to avoid some type warning while comparing **int** and **long unsigned int**:

`if (col_nr_internal >= stmt->bound_cols.size()) {`